### DISCUSSION TITLE
Color mismatch for skybox and background in FrameLayout in Android

### DISCUSSION THREAD

**Discussion by yicong123456** [google/filament#7427] [Q&A]
Just have a little issue that I used scene.skybox to set the background color for surfaceview in android: scene.skybox = Skybox.Builder().color(1.0f, 1.0f, 1.0f, 1.0f).build(engine)   . and I suppose it should be the same color as I set in xml: 
![20231213-162937](https://github.com/google/filament/assets/127708940/e3f8e14e-e92d-47ed-bad2-66c47b93594b)
however the colors were not match. 
![20231213-162136](https://github.com/google/filament/assets/127708940/04fd0a90-4d57-4f2b-9781-bba8d679e004)
  And I used FFFFFFFF for the rgba value and it should be the same value for Srgba and Linear type I think. Can someone help me out the reason of the color mismatch for the same value and thanks so much!.

**Comment by romainguy** [comment]
That's because filament works in HDR by default and applies tonemapping. Either use an inverse tonemapping function to compute your sky color, or use a linear tone mapper on the Color grading API.

**Reply by yicong123456** [reply]
Got it , thanks so much for the explanation!
